functions/pe_db_names.pp

```diff
@@ -0,0 +1,40 @@
+function peadm::pe_db_names (
+  String $pe_ver,
+) >> Array {
+  case $pe_ver {
+    # The patching service was added in 2025.0.0
+    SemVerRange('>= 2025.0.0'): {
+      [
+        'pe-activity',
+        'pe-classifier',
+        'pe-inventory',
+        'pe-orchestrator',
+        'pe-rbac',
+        'pe-hac',
+        'pe-patching',
+      ]
+    }
+
+    # The host-action-collector (hac) was added in 2023.8
+    SemVerRange('>= 2023.8.0'): {
+      [
+        'pe-activity',
+        'pe-classifier',
+        'pe-inventory',
+        'pe-orchestrator',
+        'pe-rbac',
+        'pe-hac',
+      ]
+    }
+
+    default: {
+      [
```

Comment:
I'd pull this out as a common ref hash, and just have the others + to it.

Reply:
Oh, right, yeah that's better